client/lbclient/boot/replication.js

```diff
@@ -23,17 +23,33 @@ module.exports = function(client) {
   };
 
   // setup model replication
-  function sync(cb) {
-    if (client.network.isConnected) {
-      RemoteTodo.replicate(LocalTodo, function() {
-        LocalTodo.replicate(RemoteTodo, cb);
+  var since = {push: -1, pull: -1};
+  function sync() {
+    LocalTodo.replicate(
+      RemoteTodo,
+      since.push,
+      function pushed(err, conflicts, cps) {
+        console.log('LR', arguments);
```

Comment:
Figured it out, see https://github.com/strongloop/loopback-example-offline-sync/pull/60/files#r28929070